<html xmlns:py="http://genshi.edgewall.org/" xmlns:xi="http://www.w3.org/2001/XInclude" py:strip="">

<py:def function="announcement_id">
PMASA-2012-3
</py:def>

<py:def function="announcement_date">
2012-08-09
</py:def>

<py:def function="announcement_summary">
Path disclosure due to missing library.
</py:def>

<py:def function="announcement_description">
The show_config_errors.php script does not include a library, so an error
message shows the full path of this file, leading to possible further attacks.
</py:def>

<py:def function="announcement_mitigation">
For the error messages to be displayed, php.ini's error_reporting must be set 
to E_ALL and display_errors must be On (these settings are not recommended 
on a production server in the PHP manual).
</py:def>

<py:def function="announcement_severity">
We consider this vulnerability to be non critical.
</py:def>

<py:def function="announcement_affected">
Versions 3.5.x before 3.5.2.1 are affected.
</py:def>

<py:def function="announcement_solution">
Upgrade to phpMyAdmin 3.5.2.1 or newer or apply the related patch listed below.
</py:def>

<py:def function="announcement_references">
Thanks to Edgar Galan for reporting this issue.
</py:def>

<py:def function="announcement_cve">CVE-2012-4219</py:def>

<py:def function="announcement_cwe">20 200</py:def>

<py:def function="announcement_commits">
0f0c2f1e2b3ece41cc1bb99a9931c8fcc7c917bc
</py:def>

<xi:include href="_page.tpl" />
</html>
